src/calculator.js

```diff
@@ -0,0 +1,10 @@
+const Calculator = (num1, num2) => {
+  const add = () => num1 + num2;
+  const subtract = () => num1 - num2;
+  const divide = () => num1 / num2;
+  const multiply = () => num1 * num2;
+  return {
+    add, divide, subtract, multiply,
+  };
+};
```

Comment:
- [ ] Also this is expected to be an object not a constructor function. [Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/testing_practice/calculator_requirement.png). It should be in this manner:

```
const calculator = {
add: () => {},
...
}
```